### ISSUE TITLE
Failing to write QuickFixGPS

### ISSUE THREAD

**Issue by dlenski** [dlenski/ttblue#1]
@weechatter alerted me to this issue. On August 26-27, `ttblue` is failing to write QuickFixGPS file that is `0x7970` bytes long.

```
$ ttblue -D -D # packet-by-packet output for file writing
...
7948: c2b375d30eceeca039ed35c5000001be0807262b
795c: 10890cb420d00c6700e3e20d0f440b4100000020
at file position 0x7970
fail: Resource temporarily unavailable
Failed to send QuickFixGPS update to watch.
Opening L2CAP LE connection on ATT channel:
	 src: 20:16:D8:CE:76:9B
	dest: E4:04:39:17:62:B1
```

**Comment by dlenski** [comment]
Fixed in 9044a80: `tt_write_file` wasn't actually writing the 2-byte CRC when the last packet was exactly 20 bytes  long.

**Comment by dlenski** [comment]
Fixed in f58df52: `tt_write_file` wasn't actually writing the 2-byte CRC when the last packet was exactly 20 bytes long.
